src/Authentication/Authentication/custom/Find-MgGraphPermission.ps1

```diff
@@ -15,12 +15,18 @@ Find permissions for authorization against Microsoft Graph
 
 # Import-Module -Force .\Microsoft.Graph.Authentication.psd1
 
-function Find-MgGraphPermission($search) {
```

Comment:
We should use a more standard name like `$SearchString` -- also need to capitalize parameter names.